Compilation error: unresolved imports

`emacs-ng` from git:
```
$ git log -1
commit c30d39ed9b4ada9b29fcec6233c11a6c2cc382a8 (HEAD -> master, origin/master, origin/HEAD)
    webrender: implement xw-color-values. Fixes #129.
```
`rustc`:
```
$ rustc -V
rustc 1.51.0-nightly (a62a76047 2021-01-13)
```
configured with:
```
$ ./configure --with-webrender
```
make command:
```
$ make -j 4
```
Fails with:
```
   Compiling emacsng v0.1.0 (/home/user/emacs-ng/rust_src)
error[E0432]: unresolved imports `lisp::remacs_sys::_bindgen_ty_52::ctrl_modifier`, `lisp::remacs_sys::_bindgen_ty_52::meta_modifier`, `lisp::remacs_sys::_bindgen_ty_52::shift_modifier`, `lisp::remacs_sys::_bindgen_ty_52::super_modifier`
 --> src/webrender_backend/keyboard.rs:6:26
  |
6 |         _bindgen_ty_52::{ctrl_modifier, meta_modifier, shift_modifier, super_modifier},
  |                          ^^^^^^^^^^^^^  ^^^^^^^^^^^^^  ^^^^^^^^^^^^^^  ^^^^^^^^^^^^^^ no `super_modifier` in `remacs_sys::_bindgen_ty_52`
  |                          |              |              |
  |                          |              |              no `shift_modifier` in `remacs_sys::_bindgen_ty_52`
  |                          |              no `meta_modifier` in `remacs_sys::_bindgen_ty_52`
  |                          no `ctrl_modifier` in `remacs_sys::_bindgen_ty_52`

error: aborting due to previous error

For more information about this error, try `rustc --explain E0432`.
error: could not compile `emacsng`
```

The current error:
```
   Compiling dprint-plugin-typescript v0.37.0
error[E0432]: unresolved imports `lisp::remacs_sys::_bindgen_ty_52::ctrl_modifier`, `lisp::remacs_sys::_bindgen_ty_52::meta_modifier`, `lisp::remacs_sys::_bindgen_ty_52::shift_modifier`, `lisp::remacs_sys::_bindgen_ty_52::super_modifier`
 --> src/webrender_backend/keyboard.rs:6:26
  |
6 |         _bindgen_ty_52::{ctrl_modifier, meta_modifier, shift_modifier, super_modifier},
  |                          ^^^^^^^^^^^^^  ^^^^^^^^^^^^^  ^^^^^^^^^^^^^^  ^^^^^^^^^^^^^^ no `super_modifier` in `remacs_sys::_bindgen_ty_52`
  |                          |              |              |
  |                          |              |              no `shift_modifier` in `remacs_sys::_bindgen_ty_52`
  |                          |              no `meta_modifier` in `remacs_sys::_bindgen_ty_52`
  |                          no `ctrl_modifier` in `remacs_sys::_bindgen_ty_52`

error: aborting due to previous error

For more information about this error, try `rustc --explain E0432`.
error: could not compile `emacsng`
```

--- a/rust_src/crates/lisp/src/remacs_sys.rs
+++ b/rust_src/crates/lisp/src/remacs_sys.rs
@@ -182,3 +182,20 @@ impl Clone for hash_table_test {
         }
     }
 }
+
+pub mod EmacsModifiers {
+    pub type Type = u32;
+
+    pub const up_modifier: Type = 1;
+    pub const down_modifier: Type = 2;
+    pub const drag_modifier: Type = 4;
+    pub const click_modifier: Type = 8;
+    pub const double_modifier: Type = 16;
+    pub const triple_modifier: Type = 32;
+    pub const alt_modifier: Type = 4194304;
+    pub const super_modifier: Type = 8388608;
+    pub const hyper_modifier: Type = 16777216;
+    pub const shift_modifier: Type = 33554432;
+    pub const ctrl_modifier: Type = 67108864;
+    pub const meta_modifier: Type = 134217728;
+}

--- a/rust_src/src/webrender_backend/keyboard.rs
+++ b/rust_src/src/webrender_backend/keyboard.rs
@@ -3,8 +3,9 @@ use glutin::event::{ModifiersState, VirtualKeyCode};
 use lisp::{
     lisp::LispObject,
     remacs_sys::{
-        _bindgen_ty_52::{ctrl_modifier, meta_modifier, shift_modifier, super_modifier},
-        event_kind, input_event, scroll_bar_part, Qnil,
+        event_kind, input_event, scroll_bar_part,
+        EmacsModifiers::{ctrl_modifier, meta_modifier, shift_modifier, super_modifier},
+        Qnil,
     },
 };
 

